src/containers/my-cooperations/accept-cooperation-close/AcceptCooperationClosing.tsx

```diff
@@ -38,7 +91,32 @@ const AcceptCooperationClosing: React.FC<AcceptCooperationClosureProps> = ({
       }
       icon={<ErrorOutlineRounded />}
       title={t('titles.acceptCooperationClosing')}
-    />
+    >
+      {isInputShown ? (
+        <Box sx={styles.inputBox}>
+          <Typography sx={styles.textGray}>
+            {t('cooperationDetailsPage.InputFieldLabel')}{' '}
```

Comment:
I am not sure that it is needed here
```suggestion
            {t('cooperationDetailsPage.InputFieldLabel')}
```